rubrik_cdm/data_management.py

```diff
@@ -2258,3 +2254,39 @@ def get_esxi_subnets(self, timeout=15):  # pylint: ignore
         self.log("get_esx_subnets: Retrieving the preferred subnets used to reach the ESXi hosts.")
 
         return self.get('internal', '/vmware/config/esx_subnets', timeout)
+
+    def get_all_hosts(self, timeout=15):
+            """Retrieve information for each host connected to the Rubrik cluster.
+            Arguments:
+                timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
+            Returns:
+                dict -- The result of the API call `GET /v1/host`
+            """
+
+            self.log('get_all_hosts: Getting information for each host on the Rubrik cluster.')
+            current_hosts = self.get('v1', '/host', timeout=timeout)
```

Comment:
we can just return the API call directly

`return = self.get('v1', '/host', timeout=timeout)`

Reply:
Updated code, thanks for spotting that!

New code in: cf7e991